Makefile

```diff
@@ -630,6 +638,10 @@ release-alias-tag: ## Adds the tag to the last build tag.
 release-notes: $(RELEASE_NOTES_DIR) $(RELEASE_NOTES)
 	go run ./hack/tools/release/notes.go --from=$(PREVIOUS_TAG) > $(RELEASE_NOTES_DIR)/$(RELEASE_TAG).md
 
+.PHONY: promote-images
+promote-images: $(KPROMO)
+	$(KPROMO) pr --project cluster-api --tag $(RELEASE_TAG) --reviewers "$(IMAGE_REVIEWERS)" --fork $(USER_FORK)
```

Comment:
@vincepri sorry for cancelling your lgtm, just added quotes around the reviewers to make sure it treats the whole thing as one arg:

```
make promote-images
hack/tools/bin/kpromo pr --project cluster-api --tag v1.0.0 --reviewers "@CecileRobertMichon @enxebre @fabriziopandini @vincepri" --fork CecileRobertMichon
```

Reply:
this is good to go now